src/impl/locale.js

```diff
@@ -75,22 +75,26 @@ function parseLocaleString(localeStr) {
     return [localeStr];
   } else {
     let options;
+    let selectedStr;
     const smaller = localeStr.substring(0, uIndex);
```

Comment:
You can now move this substring into the `catch`. It was only outside of it because we used it in the `return`.